service/internal/builder/exporters_builder.go

```diff
@@ -143,15 +143,15 @@ type exportersRequiredDataTypes map[config.Exporter]dataTypeRequirements
 // exportersBuilder builds exporters from config.
 type exportersBuilder struct {
 	logger    *zap.Logger
-	appInfo   component.ApplicationStartInfo
+	appInfo   component.BinaryInfo
```

Comment:
```suggestion
	binInfo   component.BinaryInfo
```

Should probably update the field name to better reflect the type.  Carry this through to any place using `appInfo`.